foodalert/static/foodalert/css/styles.css

```diff
@@ -28,3 +34,7 @@
     border-color: red;
     color: red;
 }
+
+textarea, input {
```

Comment:
It may also be preferable to scope CSS to the component it's supposed to affect. There may be a place for global CSS rules for branding, but layout styles are probably better placed in the component itself.